.gitignore

```diff
@@ -31,3 +31,4 @@ xliveless.v12.suo
 /Halo2
 /xlive/Debug
 /xlive/Release
+*.opensdf
```

Comment:
why is this being added? please remove unless necessary

Reply:
cuz its  a practically useless file in my regard :)